packages/patternfly-4/react-core/src/demos/PaginationTable.md

```diff
@@ -50,17 +70,50 @@ class ComplexPaginationTableDemo extends React.Component {
   }
 
   render() {
-    const { loading } = this.state;
+    const { loading, res } = this.state;
+    if (res.length === 0) {
+      const rows = [{
+        heightAuto: true,
+        cells: [
+          {
+            props: { colSpan: 8 },
+            title: (
+            <Bullseye>
+              <EmptyState variant={EmptyStateVariant.small}>
+                <EmptyStateIcon icon={SearchIcon} />
+                <Title headingLevel="h2" size="lg">
+                  No results found
+                </Title>
+                <EmptyStateBody>
+                  No results match the filter criteria. Remove all filters or clear all filters to show results.
+                </EmptyStateBody>
+              </EmptyState>
+            </Bullseye>
+            )
+          },
+        ]
+      }]
```

Comment:
How do users see the empty state? Do we need to edit the code to remove rows?

Reply:
we should probably add a filter to the table and if a filter is set show empty state.